aspnetcore/mvc/razor-pages/index.md

```diff
@@ -256,7 +263,7 @@ Add a *_ViewImports.cshtml* file:
 @addTagHelper "*, Microsoft.AspNetCore.Mvc.TagHelpers"
```

Comment:
Dan's preference is to remove the double quotes from the `@addTagHelper` directive.